README.md

```diff
@@ -1,18 +1,18 @@
 # TizenTube
 
-A NodeJS script to remove YouTube ads and add support for Sponsorblock for your Tizen TV (2017 and forward).
+TizenTube is a NodeJS script that enhances your Tizen TV (2017 and newer) viewing experience by removing YouTube ads and adding support for Sponsorblock.
 
 # TizenTube Installation Guide
 
 ## Prerequisites
 
 - A PC capable of running Tizen Studio, which will be used to install TizenStudio onto your TV through SDB.
-- A PC or Single Board Computer capable of running 24/7 (for ease of use) or the Android App. Your TV connects to the debugger and sends JS code to this debugger that removes video ads.
+- A PC or Single Board Computer capable of running 24/7 (for ease of use) or the Android App. When the TizenTube app starts on your TV, it initiates a debugger session using the debug <app.id> command. This allows the TV to connect to the debugger and send JavaScript code that removes video ads.
```

Comment:
> When the TizenTube app starts on your TV, it initiates a debugger session using the debug <app.id> command. This allows the TV to connect to the debugger and send JavaScript code that removes video ads.

Should be something like this:
`When you use the Launcher app or launch it from the Android app, the TV starts the TizenTube app in debugging mode, then the server connects to the debugger and sends JS code that removes ads and adds SponsorBlock.`

...Or not even included in here, but somewhere else. It shouldn't be in the requirements.